ext/node/polyfills/internal_binding/_utils.ts

```diff
@@ -27,6 +28,22 @@ export function base64ToBytes(str: string) {
   }
 }
 
+export function base64Write(
+  str: string,
+  buffer: Uint8Array,
+  offset: number = 0,
+  length?: number,
+): number {
+  length = length ?? buffer.byteLength - offset;
+  try {
+    return op_base64_write(str, buffer, offset, length);
+  } catch {
+    str = base64clean(str);
+    str = str.replaceAll("-", "+").replaceAll("_", "/");
```

Comment:
Wouldn't it be faster to replace in Rust?

Reply:
yes, that would be a seperate op. left this for now because its the slow path